linters/plugin.yaml

```diff
@@ -310,6 +310,55 @@ lint:
       extensions:
         - m
 
+    # This is the same as the lockfile type above, minus go.sum because osv-scanner does not support that
+    # TODO(Tyler): once linter ignores are supported, move this into a linter ignore
```

Comment:
alternatively: can we say osv applies to all lockfiles, and then inside default_lint.yaml ship a default ignore for go.sum / osv-scanner?

Reply:
Discussed this with Mav, I think the best approach here is to:
1. Land this (sorry I saw this comment after I merged, but compat/history shouldn't be a big obstacle)
2. Add the logic in cli to support merging ignores from the plugin repos, and then add that ignore (and all the rest) here (since it's specific to osv)
  a. One complexity is determining a merge key, since ignores don't have names, etc. Maybe we ship our default set and just add the option to append ignores.
3. Alternatively to #2, just do as you mention and ship the default_lint with a new osv-scanner ignore
4. Regardless, at that point we can ditch this filetype from the plugins repo and bump our `required_cli_version`